src/views/splash/hoc-banner/hoc-banner.jsx

```diff
@@ -0,0 +1,115 @@
+var FormattedMessage = require('react-intl').FormattedMessage;
+var injectIntl = require('react-intl').injectIntl;
+var MediaQuery = require('react-responsive');
+var React = require('react');
+
+var FlexRow = require('../../../components/flex-row/flex-row.jsx');
+var TitleBanner = require('../../../components/title-banner/title-banner.jsx');
+var TTTModal = require('../../../components/modal/ttt/modal.jsx');
+var TTTTile = require('../../../components/ttt-tile/ttt-tile.jsx');
+
+var frameless = require('../../../lib/frameless');
+var tiles = require('../../thingstotry/ttt');
+
+require('../../../components/forms/button.scss');
+require('./hoc-banner.scss');
+
+var HocBanner = injectIntl(React.createClass({
+    getInitialState: function () {
+        return {
+            currentTile: tiles[1],
+            TTTModalOpen: false
+        };
+    },
+    showTTTModal: function (tile) {
+        return this.setState({
+            currentTile: tile,
+            TTTModalOpen: true
+        });
+    },
+    hideTTTModal: function () {
+        return this.setState({TTTModalOpen: false});
+    },
+    renderTTTTiles: function () {
+        var formatMessage = this.props.intl.formatMessage;
+        
+        var tileObjects = {
+            flyTile: {
+                title: formatMessage({id: tiles[1].title}),
+                description: formatMessage({id: tiles[1].description}),
+                tutorialLoc: tiles[1].tutorialLoc,
+                activityLoc: formatMessage({id: tiles[1].activityLoc}),
+                guideLoc: formatMessage({id: tiles[1].guideLoc}),
+                thumbUrl: tiles[1].thumbUrl,
+                bannerUrl: tiles[1].bannerUrl
+            },
+            nameTile: {
+                title: formatMessage({id: tiles[0].title}),
+                description: formatMessage({id: tiles[0].description}),
+                tutorialLoc: tiles[0].tutorialLoc,
+                activityLoc: formatMessage({id: tiles[0].activityLoc}),
+                guideLoc: formatMessage({id: tiles[0].guideLoc}),
+                thumbUrl: tiles[0].thumbUrl,
+                bannerUrl: tiles[0].bannerUrl
+            },
+            musicTile: {
+                title: formatMessage({id: tiles[2].title}),
+                description: formatMessage({id: tiles[2].description}),
+                tutorialLoc: tiles[2].tutorialLoc,
+                activityLoc: formatMessage({id: tiles[2].activityLoc}),
+                guideLoc: formatMessage({id: tiles[2].guideLoc}),
+                thumbUrl: tiles[2].thumbUrl,
+                bannerUrl: tiles[2].bannerUrl
+            }
+        };
+        
+        return [
```

Comment:
I reordered the tiles in based on feedback from @ntlrsk (see https://github.com/LLK/scratch-www/issues/979#issuecomment-258456575) that put Animate Your Name first instead of Make It Fly. We should check whether this change should carry over to the splash page. If so you can just use the first three ttt tiles in order.